### ISSUE TITLE
Learning mode for autotype

### ISSUE THREAD

**Issue by chr0max** [MacPass/MacPass#851]
Hi,

if autotype is not working because the wrong window-title is in my file, I get the focus of the input, switch to macpass, search the login, start autotype or change into the correct title (some only have "login"). Maybe I´m doing it wrong, maybe I´m doing it right, then I have a nice feature idea of a self-learning. If I do this once, macpass records the window-title (or even the app!?) and do this the same way in the future. It´s a enhanced type of autotype, maybe a separate context entry like "autotype and learn" or something.

great he?

**Comment by georgesnow** [comment]
I agree and also if you had an option to expose what global autotype tried to match. I noticed in the notifications it will tell you what seems to be the window name.
Although sometimes I get weird ones like “item-0” but it works if I use it just for that app.

 And I found if I use it as tag in the entry. And set global auto type to match on tags. it works pretty well. The tags also are easier to span across entries and use multiple tags. Like work password that is used for 2 different applications. (Example: Iterm2 and fetch)
